src/plugin.js

```diff
@@ -143,7 +143,7 @@ const initPlugin = function(player, options) {
       let currentTime = player.currentTime();
 
       // playback is operating normally or has recovered
-      if (currentTime !== lastTime) {
+      if ((lastTime + 1) <= (currentTime)) {
```

Comment:
What if the user seeks forward, wouldn't this cause an issue? What about a slightly different check:
```js
if (Math.abs(currentTime - lastTime) >= 1) {
```
That should cover the case of the time change going in either direction.